tests/test_acl.py

```diff
@@ -1215,6 +1215,141 @@ def test_AclRuleIcmpV6(self, dvs, testlog):
 
         self.remove_acl_table(acl_table)
 
+    def set_admin_status(self, interface, status):
```

Comment:
Could you move these helper functions into `conftest.py` file?

Reply:
Yes, we will do it.

Reply:
I have done this in PR #943.